requirements/base.txt

```diff
@@ -12,22 +12,22 @@ coreschema==0.0.4         # via coreapi
 defusedxml==0.6.0         # via python3-openid, social-auth-core
 django-cors-headers==3.2.1  # via -r requirements/base.in
 django-extensions==2.2.8  # via -r requirements/base.in
-django-modelcluster==4.4.1  # via wagtail
+django-modelcluster==5.0.1  # via wagtail
 django-rest-swagger==2.2.0  # via -r requirements/base.in
 django-storages==1.9.1    # via -r requirements/base.in
-django-taggit==0.23.0     # via wagtail
+django-taggit==1.2.0      # via wagtail
 django-treebeard==4.3.1   # via wagtail
 django-waffle==0.20.0     # via -r requirements/base.in, edx-django-utils, edx-drf-extensions
-django==1.11.29           # via -r requirements/base.in, django-cors-headers, django-storages, django-taggit, django-treebeard, drf-jwt, edx-auth-backends, edx-django-release-util, edx-django-utils, edx-drf-extensions, rest-condition, wagtail
-djangorestframework==3.9.4  # via -r requirements/base.in, django-rest-swagger, drf-jwt, edx-drf-extensions, rest-condition, wagtail
+django==2.2.11            # via -r requirements/base.in, django-cors-headers, django-storages, django-taggit, django-treebeard, djangorestframework, drf-jwt, edx-auth-backends, edx-django-release-util, edx-django-utils, edx-drf-extensions, rest-condition, wagtail
+djangorestframework==3.11.0  # via -r requirements/base.in, django-rest-swagger, drf-jwt, edx-drf-extensions, rest-condition, wagtail
 draftjs-exporter==2.1.7   # via wagtail
 drf-jwt==1.15.0           # via edx-drf-extensions
 edx-auth-backends==3.0.2  # via -r requirements/base.in
 edx-django-release-util==0.3.6  # via -r requirements/base.in
 edx-django-utils==3.0     # via -r requirements/base.in, edx-drf-extensions, edx-rest-api-client
-edx-drf-extensions==4.0.1  # via -r requirements/base.in
+edx-drf-extensions==4.0.2  # via -r requirements/base.in
 edx-opaque-keys==2.0.1    # via edx-drf-extensions
-edx-rest-api-client==4.0.1  # via -r requirements/base.in
+edx-rest-api-client==5.0.1  # via -r requirements/base.in
```

Comment:
This has a backwards-incompatible change in timeout handling; may be best to pin it for now and review the impact of that separately.

Reply:
Pinned the version==4.0.1